boards.txt

```diff
@@ -55,15 +55,15 @@ arduino_zero_native.pid.2=0x824d
 # If the board is a 2341:824d use 2341:824d for build and set other parameters as well
 arduino_zero_native.vid.2.build.vid=0x2341
 arduino_zero_native.vid.2.build.pid=0x824d
-arduino_zero_native.vid.2.build.usb_product="Genuino Zero"
+arduino_zero_native.vid.2.build.usb_product="Arduino Zero"
 arduino_zero_native.vid.2.bootloader.file=zero/samd21_sam_ba_genuino.bin
```

Comment:
The meaning of this piece:

```
arduino_zero_native.vid.2=0x2341
arduino_zero_native.pid.2=0x824d
# If the board is a 2341:824d use 2341:824d for build and set other parameters as well
arduino_zero_native.vid.2.build.vid=0x2341
arduino_zero_native.vid.2.build.pid=0x824d
arduino_zero_native.vid.2.build.usb_product="Genuino Zero"
arduino_zero_native.vid.2.bootloader.file=zero/samd21_sam_ba_genuino.bin
```

can be summed up as:

- If you find a board with vid/pid `2341:824d` then
- build the sketch with vid/pid `2341:824d` and use usb_product `Genuino Zero` and (possibily) burn genuino bootloader.

but since there is no more need to keep `Genuino` we can just remove the extra section and leave just:

```
 arduino_zero_native.vid.2=0x2341
 arduino_zero_native.pid.2=0x824d
```

so the board is "detected" as an "Arduino Zero" and the PID 824d will be replaced with the Arduino PID 804d at the next upload.